opensrp-app/src/main/java/org/smartregister/service/HTTPAgent.java

```diff
@@ -147,9 +147,15 @@ private HttpURLConnection initializeHttp(String requestURLPath, boolean useBasic
     }
 
     public Response<String> postWithJsonResponse(String postURLPath, String jsonPayload) {
+        logResponse(postURLPath, jsonPayload);
         return post(postURLPath, jsonPayload);
     }
 
+    private void logResponse(String postURLPath, String jsonPayload) {
+        Timber.d("postURLPath: %s", postURLPath);
+        Timber.d("jsonPayLoad: %s", jsonPayload);
+    }
+
```

Comment:
Can these be combined to one once both the url and the payload

Reply:
@githengi its done! please check

Reply:
@githengi can you review please? Thanks in advance